paragraph/_wrapper.py

```diff
@@ -0,0 +1,57 @@
+"""A module loader wrapping all callables as ops on import
+
+This module defines a module finder/loader pair defining a virtual package ``paragraph.wrap``. Any installed module can be imported within this virtual
+package, resulting in all the callables defined in it to be wrapped by the ``paragraph.op`` decorator.
+
+Example:
+    >>> from paragraph.wrap.operator import add
+    >>> import paragraph as pg
+    >>> x = pg.Variable("x")
+    >>> y = add.op(x, 2)
+"""
+from importlib import import_module
+from importlib.machinery import ModuleSpec
+from typing import Callable
+
+from paragraph.types import op
+
+
+class WrappedModuleFinder:
+    """
+    Data package module loader finder. This class sits on `sys.meta_path` and returns the
+    loader it knows for a given path, if it knows a compatible loader.
```

Comment:
I suggest to rephraze the doctring to remove the informal "this class sits on ..." and better reflect the behavior of the class:

"This class is part of the machinery that allows to wrap all callables in an imported module with the ``paragraph.op`` decorator. If added to `sys.meta_path`, it is invoked before any other importer and results in``WrappedModuleLoader`` being called for all import module paths starting with `paragraph.wrap`."

Reply:
Done